raw-data/PH/phoible_inventories.tsv

```diff
@@ -4434,7 +4434,7 @@
 				"eː"				
 				"aː"				
 
-962	"igo"	"Ngomba"	"Not finished, became too confusing. To finish later"	"i"		"[ɨ]"	"In dipthongs where the second vowel is ə"	"igo_ngouagna1988.pdf"
+962	"jgo"	"Ngomba"	"Not finished, became too confusing. To finish later"	"i"		"[ɨ]"	"In dipthongs where the second vowel is ə"	"igo_ngouagna1988.pdf"
```

Comment:
the comment on this line is a bit unsettling...  is that cruft of is this entry really unfinished?

Reply:
Will address in #263